### ISSUE TITLE
Add Snowplow tracking to UI

### ISSUE THREAD

**Issue by alexanderdean** [snowplow/schema-guru#69]
Nothing sensitive should get tracked (i.e. not the JSONs or JSON schemas in any way).

**Comment by chuwy** [comment]
I haven't implemented self-describing JSON in UI. It could be interesting to know how often people use your format.
So, tracking should be enabled in one specific build (for example, if we'll decide to host this app) or always?

**Comment by alexanderdean** [comment]
Agree, we should make embedding the Snowplow tracking a build option
